src/main/java/edu/kis/vh/nursery/factory/Rhymersfactory.java

```diff
@@ -4,12 +4,12 @@
 
 	public interface Rhymersfactory {
 	
-		public DefaultCountingOutRhymer GetStandardRhymer();
+		DefaultCountingOutRhymer GetStandardRhymer();
```

Comment:
Our static analysis engine detects that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.